src/rebar_relx.erl

```diff
@@ -195,6 +199,32 @@ merge_overlays(Config) ->
     NewOverlay = lists:flatmap(fun({overlay, Overlay}) -> Overlay end, lists:reverse(Overlays)),
     [{overlay, NewOverlay} | Others].
 
+overlay_vars(Opts) ->
+    case proplists:get_value(overlay_vars, Opts) of
+        undefined ->
+            [];
+        OverlayVars ->
+            [OverlayVars]
+    end.
+
+maybe_obey_command_args(RelxConfig, Opts, Args) ->
+    lists:foldl(
+        fun({Opt, AliasOpt}, Acc) ->
```

Comment:
`AliasOpt`? Are you having `vm_args` replaced with `vm_args_src`?

They are separate args. Also, `sys_config_src` exists as an option but isn't included.

Reply:
yep, they should be mutually exclusive, rather than treating one as the priority of the other.

Reply:
Hmm alright. Now I got it. Right now, this PR is just forwarding the vm_args and sys_config to relx.

If I have a release configuration like this:

```
{relx, [
  {release, {elock, "0.1.0"}, [elock, sasl],
    [
        {dev_mode, true},
        {include_erts, false},
        {system_libs, false},

        {extended_start_script, true},
        {generate_start_script, true},

        {overlay, []}
    ]},
```
The following command will work: `rebar3 tar --system_libs $REBAR3_TARGET_SYSTEM_LIBS --include-erts $REBAR3_TARGET_INCLUDE_ERTS -n elock --sys_config config/sys-test.config --vm_args config/vm-test.args`.

On the other hand, the following release configuration will not work with the same `rebar3 tar` like the above command:

```
{relx, [
  {release, {elock, "0.1.0"}, [elock, sasl],
    [
        {sys_config, "./config/sys.config"},
        {vm_args, "./config/vm.args"},

        {dev_mode, true},
        {include_erts, false},
        {system_libs, false},

        {extended_start_script, true},
        {generate_start_script, true},

        {overlay, []}
    ]},
```